test/e2e/case2_config_deployment_test.go

```diff
@@ -99,7 +99,7 @@ var _ = Describe("Test config-policy-controller deployment", func() {
 
 			By(logPrefix +
 				"removing the config-policy-controller deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=30s")
+			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=90s")
 			deploy := GetWithTimeout(
 				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 180,
```

Comment:
I made progress locally by adjusting this timeout for the case2 failures which I see you are still running into.

```suggestion
				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 360,
```